tests/build/chains.go

```diff
@@ -250,17 +273,21 @@ var _ = framework.ChainsSuiteDescribe("Tekton Chains E2E tests", func() {
 					"MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAENZxkE/d0fKvJ51dXHQmxXaRMTtVz\n" +
 					"BQWcmJD/7pcMDEmBcmk8O1yUPIiFj5TMZqabjS9CQQN+jKHG+Bfi0BYlHg==\n" +
 					"-----END PUBLIC KEY-----")
+				g.GinkgoWriter.Println("Create an invalid public signing key")
 				Expect(kubeController.CreateOrUpdateSigningSecret(publicKey, secretName, namespace)).To(Succeed())
 				generator.PublicSecret = fmt.Sprintf("k8s://%s/%s", namespace, secretName)
 
 				pr, err := kubeController.RunPipeline(generator, pipelineRunTimeout)
+				g.GinkgoWriter.Printf("Running pipeline \"%s\"\n", pr.Name)
 				Expect(err).NotTo(HaveOccurred())
+				g.GinkgoWriter.Printf("Waiting for pipeline \"%s\" to finish\n", pr.Name)
 				err = kubeController.WatchPipelineRun(pr.Name, pipelineRunTimeout)
```

Comment:
Is there a way to call `g.GinkgoWriter.Printf` from `kubeController.WatchPipelineRun` and the other helper methods? It would avoid some repetition.